plugins/inputs/ah_wireless/ah_wireless.go

```diff
@@ -0,0 +1,942 @@
+package ah_wireless
+
+import (
+	"log"
+	"os"
+	"bytes"
+	"strconv"
+	"syscall"
+	"strings"
+	"sync"
+	"time"
+	"os/exec"
+	"fmt"
+	"unsafe"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"golang.org/x/sys/unix"
+)
+
+var (
+	offsetsMutex = new(sync.Mutex)
+	newLineByte  = []byte("\n")
+)
+
+
+type Ah_wireless struct {
+	fd     int
+	fe_fd	uintptr
+	intf_m  map[string]map[string]string
+	arp_m	map[string]string
+	Ifname	[]string	`toml:"ifname"`
+	closed chan struct{}
+	numclient int
+}
+
+
+func wl_ioctl(fd int, op, argp uintptr) error {
```

Comment:
wl_ioctl is the same as fe_ioctl. Consider unifying them as one function.

Reply:
@zhchen-extr , Yes. That was a good point. I have changed it now.